lib/orbit_web/controllers/sign_in_controller.ex

```diff
@@ -0,0 +1,36 @@
+defmodule OrbitWeb.SignInController do
+  alias Orbit.Employee
+  alias Orbit.OperatorSignIn
+  alias OrbitWeb.Auth.Auth
+
+  use OrbitWeb, :controller
+  import Ecto.Query
+
+  alias Orbit.Repo
+
+  @spec submit(Plug.Conn.t(), map()) :: Plug.Conn.t()
+  def submit(conn, %{
+        "signed_in_employee_badge" => signed_in_employee_badge,
+        "signed_in_at" => signed_in_at,
+        "line" => line,
+        "method" => method
+      }) do
+    signed_in_at = DateTime.from_unix!(signed_in_at)
+
+    # ⚠️ TODO: remove before merge
+    :timer.sleep(1000)
+
+    %OperatorSignIn{
+      signed_in_employee:
+        Repo.one!(from(e in Employee, where: e.badge_number == ^signed_in_employee_badge)),
```

Comment:
**suggestion:** could we give a more specific response code if no operator for the corresponding badge is found? I think a 404 would be semantically appropriate but I'd have to read a bit more to confirm.

Reply:
Added a test; already works that way
b297dd1

Reply:
Oh, right, I forgot that Ecto + Phoenix does some fancy stuff to turn Ecto exceptions into relevant HTTP response codes that usually make sense. But thanks for adding that test, that's good to have!